src/Umbraco.Core/ContentExtensions.cs

```diff
@@ -137,7 +137,8 @@ internal static bool HasChildren(IContentBase content, ServiceContext services)
         /// <returns></returns>
         public static IEnumerable<Property> GetNonGroupedProperties(this IContentBase content)
```

Comment:
This can easily have `IContentTypeBaseServiceProvider` passed to it, i'll fix up